.github/workflows/tests.yaml

```diff
@@ -76,7 +77,14 @@ jobs:
         uses: actions/cache@v4
         with:
           path: /home/runner/.nox
-          key: ${{ runner.os }}-python-${{steps.setup-python.outputs.python-version }}-${{ hashFiles('noxfile.py') }}
+          key: ${{ runner.os }}-python-${{matrix.pyv}}-${{ hashFiles('noxfile.py') }}-${{matrix.vllm_version}}
+
+      - name: hf hub cache
+        id: hf-cache
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/huggingface/hub
+          key: ${{ runner.os }}
```

Comment:
Is the HF Cache specific to the OS or can it be shared across OSes?

Reply:
I believe it can be shared across OSes, but I just needed a dummy key because the cache complains otherwise. I don't think we're even interested in running this workflow for multiple OSes.